dali/aux/optical_flow/turing_of/optical_flow_buffer.h

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_AUX_OPTICAL_FLOW_TURING_OF_OPTICAL_FLOW_BUFFER_H_
+#define DALI_AUX_OPTICAL_FLOW_TURING_OF_OPTICAL_FLOW_BUFFER_H_
+
+#include <third_party/turing_of/nvOpticalFlowCuda.h>
```

Comment:
```suggestion
#include "third_party/turing_of/nvOpticalFlowCuda.h"
```

Reply:
Hmm, shouldn't we add those headers with CMake, so they will be visible during build as library headers?

Like with gtest, we dont:
`#include "third_party/gtest/include/something"` but `#include <gtest/something>`

Reply:
For `gtest` we use `#inlcude <gtest/gtest.h>`, so I suppose I should use the same routine for OF, since it's also a submodule

Reply:
> Hmm, shouldn't we add those headers with CMake, so they will be visible during build as library headers?
> 
> Like with gtest, we dont:
> `#include "third_party/gtest/include/something"` but `#include <gtest/something>`

I'd need to extend `Dependencies.cmake` file, which seems like an overkill for 2 header files

Reply:
I'd do the following:
1. Add third_party to include path (probably it's already there if the gtest include works)
2. ```#include <turing_of/nvOpticalFlowCuda.h>```
But that being said, shouldn't those header come with the cuda version somewhere?
Is this `third_party/turing_of` a temporary location?

Reply:
That would be about a two lines change in Dependencies.cmake, so I think it's fine. Especially as it is supposed to handle the third_party dependencies.